web/alt_text_manager.js

```diff
@@ -157,7 +181,20 @@ class AltTextManager {
     }
   }
 
+  #cancel() {
+    this.#uiManager?.reportTelemetry({
```

Comment:
Here, and elsewhere in this file, since the `eventBus` is already available can't we reduce (a tiny bit of needless) indirection by just dispatching the event directly?
```suggestion
    this.#eventBus.dispatch("reporttelemetry", {
```

Reply:
Yep I can do that but then the eventBus in the uiManager has to be public.
And from my pov, the way the data are passed to the telemetry is a detail of implementation (uiManager::reportTelemetry could do it in an other way).
That said, I'm ok to change, just tell me.

Reply:
Well, my point was just that the `AltTextManager` class already has *direct* access to an eventBus-instance which is why I'm suggesting that we use that here :-)

> Yep I can do that but then the eventBus in the uiManager has to be public.

We could perhaps just change `uiManager.#eventBus` into e.g. `uiManager._eventBus`, and get rid of the new `uiManager::reportTelemetry` method that way, however that could be deferred to a follow-up to reduce scope here.